openapi/components/schemas/Storefront/CustomerJWT.yaml

```
@@ -0,0 +1,23 @@
+type: object
+properties:
+  id:
+    description: The session identifier string
+    allOf:
+      - $ref: ../ResourceId.yaml
+    readOnly: true
+  token:
+    description: The session's token used for authentication
+    type: string
+    readOnly: true
+  createdTime:
+    description: Session created time
+    allOf:
+        - $ref: ../ServerTimestamp.yaml
+  updatedTime:
+    description: Session updated time
+    allOf:
+      - $ref: ../ServerTimestamp.yaml
+  expiredTime:
+    description: Session expired time. Defaults to one hour
```

When you use sentences and punctuation, then you must use punctuation.
```suggestion
    description: Session expired time. Defaults to one hour.
```